site/docs/spec.md

```diff
@@ -581,10 +596,11 @@ Table metadata consists of the following fields:
 | _optional_ | _optional_ | **`metadata-log`**| A list (optional) of timestamp and metadata file location pairs that encodes changes to the previous metadata files for the table. Each time a new metadata file is created, a new entry of the previous metadata file location should be added to the list. Tables can be configured to remove oldest metadata log entries and keep a fixed-size log of the most recent entries after a commit. |
 | _optional_ | _required_ | **`sort-orders`**| A list of sort orders, stored as full sort order objects. |
 | _optional_ | _required_ | **`default-sort-order-id`**| Default sort order id of the table. Note that this could be used by writers, but is not used when reading because reads use the specs stored in manifest files. |
+|            | _optional_ | **`refs`** | A list of snapshot references, stored as full snapshot reference objects. |
+|            | _optional_ | **`current-branch`** | The name of the current branch. If not specified, it defaults to the `main` branch that starts with the table creation commit. | 
```

Comment:
Do we need to keep each branch's latest snapshot id here ? I wonder how do we support query on each branch or tag ?

Reply:
the `refs` always record the latest snapshot ID of a branch, query of branch and tag will use that information.

Reply:
As I noted above, I think that we should remove this. The "current branch" must always be `main`.

That simplifies a couple things in the spec:
* If `refs` is present, it must contain a `main` branch
* The `main` branch reference never expires
* No need for `current-branch`
* `current-snapshot-id` must always match the snapshot ID of the `main` branch